packages/components/package.json

```
@@ -25,8 +25,10 @@
     "clean": "rimraf dist",
     "cleanAll": "rimraf dist && rimraf node_modules",
     "prepublishOnly": "yarn install && cross-env WEBPACK_STATS=errors-only yarn build",
-    "test": "cross-env NODE_ENV=test jest --maxWorkers=6 --silent",
-    "test-ci": "cross-env NODE_ENV=test jest --ci --silent",
+    "test": "npm run test-react ; npm run test-enzyme",
+    "test-enzyme": "cross-env NODE_ENV=test jest --config jest.enzyme.config.js --maxWorkers=6 --silent",
+    "test-react": "cross-env NODE_ENV=test jest --config jest.react.config.js",
```

should either of the following flags be set for `test-react` as well? `--maxWorkers=6 --silent`

---

I've added `--maxWorkers=6` so it doesn't over consume processors on dev machines. I've left `--silent` off in the hopes that we can squash failures/warnings for these from the outset.